### DISCUSSION TITLE
Onedrive won't launch because of a dlang update?

### DISCUSSION THREAD

**Discussion by jayschuster** [abraunegg/onedrive#2323] [Q&A] [answered]
Hi, it seems that the client doesn't work with the latest version of the d runtime that I have installed. I've reinstalled the client, ldc, and liblphobos, but onedrive always fails to launch with the same error (in attached image.) I tried making symlinks from the latest version of the files (libphpbos2-ldc-shared.so.101), but that just ended up causing more errors later on. No idea how to fix this as I've never used D before, so any help would be appreciated! I also assume I could downgrade ldc and liblphobos on my system, but I would prefer not to, as downgrades are not recommended by the arch wiki.
![image](https://user-images.githubusercontent.com/69011306/222048492-e0608df0-2873-4cdd-86a8-06be11c69c35.png)

**Comment by abraunegg** [comment, marked as answer]
@spokrr 
What Operating System | Linux distribution is this?

Are you following the *correct* installation process for that distribution? If Ubuntu | Debian or anything based on them - are you following: https://github.com/abraunegg/onedrive/blob/master/docs/ubuntu-package-install.md

**Reply by jayschuster** [reply]
Its Arch, and I installed onedrive-abraunegg from AUR.

**Reply by abraunegg** [reply]
In that case you need to raise your query with the AUR package maintainer as the package needs to be rebuilt. 

Raise an issue here: https://aur.archlinux.org/packages/onedrive-abraunegg

I have zero control or capability to assist here - sorry.

**Reply by jayschuster** [reply]
Thats alright - thanks for the help!
